extensions/wikia/Scribe/ScribeEventProducerController.class.php

```diff
@@ -39,17 +31,6 @@ static public function onSaveComplete( &$oPage, &$oUser, $text, $summary, $minor
 
  		$oScribeProducer = new ScribeEventProducer( $key, $is_archive );
 		if ( is_object( $oScribeProducer ) ) {
-			/**
-			 * Ugly fix! TODO
-			 * See the description above.
-			 */
-			$oTitle = $oPage->getTitle();
-			if ( $oTitle->getNamespace() == NS_FILE ) {
-				self::$oPage = $oPage;
-				self::$oUser = $oUser;
-				self::$oRevision = $oRevision;
-			}
-
 			if ( $oScribeProducer->buildEditPackage( $oPage, $oUser, $oRevision ) ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Merge this if statement with the enclosing one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1066)